docs/users/sync-data.mdx

```diff
@@ -29,26 +24,28 @@ To enable webhooks, go to the [Webhooks](https://dashboard.clerk.com/last-active
 
 You'll be presented with a form where you can specify the URL of your backend endpoint. This is the URL where Clerk will send the webhook events. You can also specify the events you want to receive. For example, if you only want to receive events related to users, you can select the **user** option.
 
+<Callout>
+If you are developing on your localhost, you will need to expose your endpoint to the internet to work with webhooks. See the [Testing the webhook](#testing-the-webhook) section for information.
+</Callout>
+
 ![Add endpoint form](/docs/images/users/guides/sync-backend/webhook-form.png)
 
 Once you click the **Create** button, you'll be presented with your webhook endpoint dashboard. Here you can see the URL of your endpoint and the events you selected and you can also test your endpoint.
 
-### Setup your backend endpoint
+### Signing secret
```

Comment:
Can you change this to be action-oriented? Like "Set up your signing secret". (As per our styleguide!)

Reply:
Changed to "Adding your Signing Secret to your .env.local file" but question about this. In the Dashboard is 'Signing Secret'. The style for the docs is 'Signing secret'. Where should I land on this?

1. Adding your Signing Secret to your .env.local file
2. Adding your 'Signing Secret' to your .env.local file
3. Adding your Signing aecret to your .env.local file
4. or something else? Currently its number 1.

Reply:
if we call it Signing Secret in the Clerk Dashboard, let's update the entire page to use Signing Secret all throughout. Don't forget to make `.env.local` a code snippet